### PR TITLE
add product card

### PR DESCRIPTION
**Pull request by a-kovalov**
- [DEMO LINK](https://a-kovalov.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://a-kovalov.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/styles/blocks/card.css [hunk 1]
```diff
@@ -0,0 +1,119 @@
+/* #region card block */
+
+.card {
+  box-sizing: border-box;
```
Comment (etojeDenys) [inline src/styles/blocks/card.css:4]: move to * selector

File: src/styles/blocks/card.css [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+  align-items: flex-start;
+  width: 200px;
+  background: #ffff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  --text-main-accent-color: #060b35;
+  --text-secondary-color: #616070;
+  --button-main-color: #00acdc;
```
Comment (etojeDenys) [inline src/styles/blocks/card.css:16]: move to ::root selector

File: src/styles/blocks/card.css [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+  margin-bottom: 19px;
+}
+
+.reviews--position {
+  margin-left: 22px;
+}
+
+.reviews {
+  width: 53px;
+  font-weight: 400;
```
Comment (etojeDenys) [inline src/styles/blocks/card.css:63]: redundant

File: src/styles/blocks/stars.css [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+.stars--4 .stars__star {
+  display: flex;
+  background-image: url("/images/star-active.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+}
+
```
Comment (etojeDenys) [inline src/styles/blocks/stars.css:19]: reuse styles from `stars` task

**Review by etojeDenys — CHANGES_REQUESTED** [review]

**Comment by a-kovalov** [comment]
- [DEMO LINK](https://a-kovalov.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://a-kovalov.github.io/layout_product-cards/report/html_report/)
Fixed version of Product Cards.

**Comment by a-kovalov** [comment]
- [DEMO LINK](https://a-kovalov.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://a-kovalov.github.io/layout_product-cards/report/html_report/)
Main and final version: **_fixed hover main version product-cards_** - please ignore _fixed product-cards._

**Review by VitaliyBondarenko1982 — APPROVED** [review]

**Pull request closed without merging**